packages/openapi-ts/src/plugins/@hey-api/client-core/client.ts

```diff
@@ -69,6 +69,14 @@ export const createClient: PluginHandler = ({ context, plugin }) => {
         key: getClientBaseUrlKey(context.config),
         value: resolvedBaseUrl,
       });
+    } else if (resolvedBaseUrl !== '/' && resolvedBaseUrl.startsWith('/')) {
```

Comment:
I actually don't know but is something like `./foo` a legal value? If so we'd want to handle that as well